aws/v19.0.0/announcement.md

```diff
@@ -0,0 +1,7 @@
+**Workload cluster release v19.0.0 for AWS is available**. This release introduces Kubernetes 1.24, replaces the use of AWS CNI with Cilium and upgrades most components. We are also deprecating and removing use of `kiam` by default in favor of `IAM Roles for Service Accounts (IRSA)`. Please check [IAM permissions requirements](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/master/CHANGELOG.md#330---2023-05-11). Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v19.0.0/).
```

Comment:
is docs entry merged? I get 404

Reply:
we need to merge before we can update docs

Reply:
ok I see https://github.com/giantswarm/docs/pull/1813/files